_life/money.md

```diff
@@ -79,75 +71,74 @@ Do not expect good yields from non-investment bank accounts, even the savings on
 
 While you technically can cash checks, banks will charge large fees for doing so, as virtually no one in Switzerland uses checks any more.
 
+
 ## Internships in the US
 
-The most likely complication in your personal finances will be if you do an
-internship in the US. Luckily, this guide has you covered with [banking, taxes, and
-other advice](/life/money-americans#internships-in-the-us-for-non-citizens) for that
-circumstance.
+The most likely complication in your personal finances will be if you do an internship in the US.
+Luckily, this guide has you covered with [banking, taxes, and other advice](/life/money-americans#internships-in-the-us-for-non-citizens) for that circumstance.
+
 
 ## Retirement
 
 Switzerland has a system based on three pillars: shared mandatory, personal mandatory, and personal optional.
 
-### First pillar
-
-Mandatory retirement savings, taken out of the salary, as a global pool for everyone. Called "AVS". Equivalent to "social security".
-
-### Second pillar
-
-Mandatory retirement savings, taken out of the salary, for yourself. The provider is always [Publica](https://publica.ch/) for EPFL employees.
-Money is blocked until retirement or "permanently" leaving the country.
-
-### Third pillar
+The first pillar is mandatory retirement savings, taken out of the salary, as a global pool for everyone. Called "AVS", equivalent to "social security".
```

Comment:
Well, it's not "global", is it, if it's only for Switzerland. Also the second sentence is not a sentence.